src/oracle11_fmt_plug.c

```diff
@@ -67,12 +67,15 @@ john_register_one(&fmt_oracle11);
 #include "misc.h"
 #include "common.h"
 #include "formats.h"
+#include "unicode.h"
```

Comment:
I assume you added this to get `strupr()` but it actually sits in `jumbo.h`. This works as well because `unicode.h` in turn sources that header. It's not very important but in case you end up doing any other change, you could change it.

Reply:
Yes, i added `unicode.h` because i was previously using `enc_strupper()` but you're right, i could change that back to `jumbo.h`